osquery/distributed/plugins/tls.cpp

```diff
@@ -8,6 +8,9 @@
  *
  */
 
+// This must be here to prevent a WinSock.h exists error
+#include "osquery/remote/transports/tls.h"
```

Comment:
Do you mind surrounding this with:
```cpp
// clang-format off
// This must be here to prevent a WinSock.h exists error
#include "osquery/remote/transports/tls.h"
// clang-format on
```